Changed extract for non-camera type in ReceiverClass.

Popfizz013 opened the pull request:
Corrected a mistake in the receiver class which had offset being extracted from the packets which were not of type camera.

AniMB commented on ground_station/backend/receiver.py:
```diff
@@ -60,45 +72,45 @@ def __repr__(self):
         Passes payload to the application layer according to the address field.
         '''
-        if self.address is None:
+        if self.payload is None:
```
shouldnt they be payload_type

ABhiShek1705m replied:
Yes, change self.address -> self.payload_type

AniMB reviewed (changes requested):
one minor thing

AniMB commented:
Make the change whenever possible

Popfizz013 commented:
Added the changes in the payload_type according to what we discussed Req_Retransmission -> switch, error_crc -> error_peripherals. also fixed some typos and added comments

AniMB reviewed (approved):
looks good
